File: yongkeun/3week/문자열안에문자열.js
function solution(str1, str2) {
  //includes는 문자열 안에 포함하는지 확인하는 메소드이다.
  //str2 문자열이 str1안에 포함되어있다면 true를 반환한다.
  if(str1.includes(str2)){
      return 1;
  }
  else {
      return 2;
  }
}

File: gang/week3/문자열 안에 문자열.java
class Solution {
    public int solution(String str1, String str2) {
        if(str1.contains(str2)) {   //contains함수를 사용하여 str1에 str2가 있는지 확인
            return 1;
        }
        else {
            return 2;
        }
    }
}